### PR TITLE
Move some more pipelines from citest to official repo

### PR DESCRIPTION
**Pull request by coolo**

### CODE REVIEW

File: gocd/staging.bot.gocd.yaml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        jobs:
+          Run:
+            resources:
+            - repo-checker
+            tasks:
+            - script: |-
+                cd scripts
+                ./staging-installcheck.py -A https://api.opensuse.org -p Leap:15.1
+    SBot.Regular.Leap.15.1:
+      group: Factory.pkglistgen
```
Comment (nilxam) [inline gocd/staging.bot.gocd.yaml]: I thought it should be Leap.pkglistgen group?

File: gocd/staging.bot.gocd.yaml [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+      - Run:
+          approval:
+            type: manual
+          jobs:
+            Run:
+              timeout: 0
+              resources:
+              - staging-bot
+              tasks:
+              - script: osc -A https://api.opensuse.org staging -p Leap:15.1 rebuild
```
Comment (nilxam) [inline gocd/staging.bot.gocd.yaml]: '-p' parameter won't working with simplified 'Leap:15.1' IIRC, it works with '-p Factory' though.

Comment (coolo) [reply gocd/staging.bot.gocd.yaml]: yeah, that's a bug in the erb

File: gocd/staging.bot.gocd.yaml (hunk 1 above)
Comment (coolo) [reply gocd/staging.bot.gocd.yaml]: that's true

**Comment by coolo** [comment]
I rather move the others than adding an if here :)

**Pull request merged**